index.js

```diff
@@ -238,57 +232,36 @@ function InigoPlugin(config) {
       return
     }
 
-    // Create inigo query
-    const query = instance.newQuery(requestContext.request.query);
-
-    let auth = requestContext.context?.inigo?.jwt;
-
-    // Create jwt from auth object
-    if (requestContext.context?.inigo?.ctx !== undefined) {
-      auth = jwt.sign(requestContext.context.inigo.ctx, null, { algorithm: "none" });
-    }
+    // Create inigo request
+    const req = instance.newRequest({
+      query: requestContext.request.query,
+      operationName: requestContext.request.operationName,
+      variables: requestContext.request.variables,
+    });
 
     // Process request
-    const processed = query.processRequest(auth);
+    const processed = req.processRequest(requestContext.request.http.headers);
     requestContext.inigo = { processed };
 
-    // Create request context, for storing blocked status
-    if (requestContext.context.inigo === undefined) {
-      requestContext.context.inigo = { blocked: false };
-    }
-
-    // is an introspection
-    if (processed?.request.data?.__schema != undefined) {
-      requestContext.request = { http: requestContext.http };  // remove request from pipeline
-      requestContext.context.inigo.blocked = true; // set blocked state
-      return { willSendResponse(respContext) {
-          setResponse(respContext, processed.request );
-          query.ingest();
-        }
-      }
-    }
-
-    // If request is blocked
-    if (processed?.result.status == "BLOCKED") {
+    if (processed?.response != null) {
       requestContext.request = { http: requestContext.http };  // remove request from pipeline
-      requestContext.context.inigo.blocked = true; // set blocked state
```

Comment:
Restore this line, make sure it's defined where it meets the condition.

Reply:
we do not have `blocked` logic anymore

Reply:
that part is public